recipes/cppitertools/all/conanfile.py

```diff
@@ -47,19 +50,15 @@ def validate(self):
         # Exclude compilers not supported by cppitertools
         if compiler_version < minimal_version[compiler]:
             raise ConanInvalidConfiguration("{} requires a compiler that supports at least C++17. {} {} is not".format(
-                self.name, compiler, tools.Version(self.settings.compiler.version.value)))
+                self.name, compiler, Version(self.settings.compiler.version.value)))
 
     def requirements(self):
         if self.options.zip_longest:
             self.requires('boost/1.75.0')
 
     def package(self):
-        self.copy("*.hpp", dst=os.path.join("include", "cppitertools"), src=self._source_subfolder, excludes=('examples/**', 'test/**'))
-        self.copy("LICENSE.md", dst="licenses", src=self._source_subfolder)
-
-    def package_info(self):
-        self.cpp_info.names["cmake_find_package"] = "cppitertools"
-        self.cpp_info.names["cmake_find_package_multi"] = "cppitertools"
+        copy(self, "*.hpp", src=os.path.join(self.source_folder, self._source_subfolder), dst=os.path.join(self.package_folder,"include", "cppitertools"), excludes=('examples/**', 'test/**'))
+        copy(self, "LICENSE.md", src=os.path.join(self.source_folder, self._source_subfolder), dst=os.path.join(self.package_folder,"licenses"))
 
```

Comment:
```suggestion
    def package_info(self):
        self.cpp_info.set_property("cmake_file_name", "cppitertools")
        self.cpp_info.set_property("cmake_target_name", "cppitertools::cppitertools")
```

Keep this information please, even if it matches default behavior of CMakeDeps, it gives a hint to maintainers that upstream provides CMake config file and recipe properly mimics its name and targets.